src/main/java/com/CezaryZal/manager/filters/validator/RoleValidator.java

```diff
@@ -0,0 +1,31 @@
+package com.CezaryZal.manager.filters.validator;
+
+import com.CezaryZal.exceptions.IncorrectRoleException;
+import com.CezaryZal.exceptions.NullInputException;
+import org.springframework.stereotype.Component;
+
+import java.util.Arrays;
+import java.util.List;
+
+@Component
+public class RoleValidator {
+
+    List<String> listRoles = Arrays.asList("ADMIN", "USER", "VIEWER");
+
+    void validRole(String role){
+        throwIfRoleIsNull(role);
+        throwIfRoleIsIncorrect(role);
+    }
+
+    private void throwIfRoleIsNull(String role){
```

Comment:
nazywalbym te metody bardziej valid.... albo check....

Reply:
wydaje mi się że to trafniejsze, ewentualnie dopisać exception throwExceprionIfRolesIsNull.
Rzucamy tutaj  wyjątek, nic nie zwracamy